File: app/src/main/java/social/entourage/android/map/entourage/CreateEntourageModule.java
package social.entourage.android.map.entourage;

import dagger.Module;
import dagger.Provides;

/**
 * Created by mihaiionescu on 28/04/16.
 */

@Module
public class CreateEntourageModule {

    private final CreateEntourageFragment fragment;


    public CreateEntourageModule(final CreateEntourageFragment fragment) {
        this.fragment = fragment;
    }

    @Provides
    public CreateEntourageFragment providesCreateEntourageFragment() {
        return this.fragment;
    };
}

File: app/src/main/java/social/entourage/android/entourage/create/CreateEntourageModule.kt
package social.entourage.android.entourage.create

import dagger.Module
import dagger.Provides

/**
 * Created by mihaiionescu on 28/04/16.
 */
@Module
class CreateEntourageModule(private val fragment: CreateEntourageFragment) {
    @Provides
    fun providesCreateEntourageFragment(): CreateEntourageFragment {
        return fragment
    }

}